.sops.yaml

```diff
@@ -0,0 +1,3 @@
+---
+creation_rules:
+  - pgp: "B6125B16B0DD59F34D6975FBF885927FDA9C48E2"
```

Comment:
its a gpg id - the secrete is the private part of the gpg key, which you either have already or will get so you can decrypt some of the provisioning stuff that _is_ secret.

Reply:
👍

Reply:
in our standardized set up, we secure solr with a simple user/password. The big reason for this the collection and config apis are not available unless you have password enabled. You'll see we use solr cloud here, mostly because we need it for many of our projects. This file configures solr with a simple solr:SolrRocks user/password pair for development.